src/core/cm/launcher/instancemanager.hpp

```diff
@@ -136,13 +147,14 @@ class InstanceManager {
     static constexpr auto cRemovePeriod = Time::cDay;
 
     Config                         mConfig;
-    StorageItf*                    mStorage           = nullptr;
-    storagestate::StorageStateItf* mStorageState      = nullptr;
-    ImageInfoProviderItf*          mImageInfoProvider = nullptr;
-
-    Timer   mCleanInstancesTimer;
-    Timer   mInitTimer;
-    UIDPool mUIDPool;
+    StorageItf*                    mStorage      = nullptr;
```

Comment:
Update members initializations
```suggestion
    StorageItf*                    mStorage      {};
```